packages/studio-web/src/app/upload/upload.component.html

```diff
@@ -72,6 +72,7 @@ <h2 i18n="Title for upload box" class="subtitle">Select an audio file</h2>
                 <div *ngIf="inputMethod.audio === 'mic'" class="mb-0 mb-md-3">
                     <div class="row">
                         <h2 i18n="Title for record box" class="subtitle">Record your voice here</h2>
+                        <p i18n="Warning for record box">Best suited for short tests.</p>
```

Comment:
I don't think this is the appropriate place for this. Maybe put it as a tooltip or as part of the shepherd tour. I find it confusing to have this as a `p` element above the button.

Reply:
I agree, it looks out of place

Reply:
Yeah, I guess I'll take it out. I don't like it as a tooltip either:
![image](https://user-images.githubusercontent.com/33264708/219132936-d9cf4fc6-3643-4ee7-8d19-363de6e275d5.png)
and this is no better:
![image](https://user-images.githubusercontent.com/33264708/219133104-e41b7bc1-3cf7-4f0d-bcf1-a1958a3caeeb.png)